app/models/order_item.rb

```diff
@@ -144,8 +142,8 @@ def consignment_product_availability
     if !ct.nil? && !ct.empty?
       qty = ct.sum(:quantity)
     elsif !product.lots.empty? && product.lots.sum(:quantity) > 0
-      qty = product.lots_by_expiration.available_specific(Time.current.end_of_minute, market_id, organization_id).sum(:quantity)
-      qty += product.lots_by_expiration.available_general(Time.current.end_of_minute).sum(:quantity)
+      qty = product.lots.available_specific(Time.current.end_of_minute, market_id, organization_id).sum(:quantity)
+      qty += product.lots.available_general(Time.current.end_of_minute).sum(:quantity)
```

Comment:
Can explain change from `lots_by_expiration` to `lots`?

Reply:
`lots_by_expiration` orders the result set, we don't care about order here and it confuses what is going on so I changed it to just `lots`.